apps/advisor/pkg/app/checkregistry/checkregistry.go

```diff
@@ -57,3 +57,8 @@ func (s *Service) Checks() []checks.Check {
 		),
 	}
 }
+
+type SpecificConfig struct {
```

Comment:
`SpecificConfig` is a bit of a weird name IMO, unless it's some convention? What about `ChecksConfig` maybe? Or even more generic like `Config` but put it in the `checks` package?

Reply:
👍  I cannot move this to the `checks` package since it'd incur in a cyclic dependency.